doc/manpage-newsboat.asciidoc

```diff
@@ -150,4 +150,4 @@ include::chapter-environment-variables.asciidoc[]
 
 == SEE ALSO
 
-podboat(1)
+Podboat(1)
```

Comment:
Manpage names are command names, not "product" names, so please revert this change.